feature-libs/cart/saved-cart/core/store/effects/saved-cart.effect.ts

```diff
@@ -125,14 +125,13 @@ export class SavedCartEffects {
               },
               GlobalMessageType.MSG_TYPE_CONFIRMATION
             );
-
             return [
               ...actions,
-              new CartActions.SetActiveCartId(cartId),
               new CartActions.LoadCartSuccess({
```

Comment:
To reviewer: We should not set 'active cart id` before we set `active cart data`, otherwise, there is a moment that the active cart data is incorrect. In this `restore saved cart`, we set `active cart id` first, it points to a `saved cart`. so, in this moment, the active cart becomes saved cart.